### ISSUE TITLE
Remove unused branches master and rc3.8.0

### ISSUE THREAD

**Issue by maehne** [logisim-evolution/logisim-evolution#2048]
We still have the unused branches `master` and `rc3.8.0`. I would like to suggest deleting [them](https://github.com/logisim-evolution/logisim-evolution/branches/all).

**Comment by davidhutchens** [comment]
I agree that these are no longer needed. I was about to suggest deleting master since its existence leads to some confusion.

But having a branch for the latest release is useful. We should also create one for 3.9.

**Comment by maehne** [comment]
IMHO, I don’t see a need for a branch for the current version, as there is already a tag `v3.9.0` in place for reference. It would only become necessary in case we need to apply hot fixes for some urgent issues. If `main` shouldn’t be in a state for release when that becomes necessary, then we can always create it quickly.

**Comment by davidhutchens** [comment]
I agree with @maehne that keeping a version branch is not necessary.

rc3.8.0 still exists and should be removed.

**Comment by maehne** [comment]
I have deleted the stale branch `rc3.8.0`, as we don't support older releases due to resource constraints.
